### DISCUSSION TITLE
Possibility to display only 1 page/slide?

### DISCUSSION THREAD

**Discussion by ghost** [xournalpp/xournalpp#3631] [Q&A]
Hello,
I'm using Xournal ++ on my surface go and I wanted to ask if it is possible to only display 1 page when annotating a PDF? Right now it is pretty difficult, especially when zooming in....

Kind regards,

Mammothsaver

**Comment by Technius** [comment]
To show only one page at a time, you can enable "Presentation Mode" near the zoom settings:

![image](https://user-images.githubusercontent.com/1066652/152656256-9fa74aca-b6bd-4920-b419-979a4a37212a.png)

**Reply by michaelcadilhac** [reply]
Am I missing something?  This is in presentation mode:

<img width="2074" height="1489" alt="image" src="https://github.com/user-attachments/assets/9ede08bf-1717-4f04-a33e-66682e923c40" />

Multiple pages are seen.  This is disrupting because going forward, the slides are at the bottom of the screen, going backward they are at the top, while the natural position would be in the middle.
